wandb/plot/line_series.py

```diff
@@ -84,3 +89,15 @@ def line_series(
         {"step": "step", "lineKey": "lineKey", "lineVal": "lineVal"},
         {"title": title, "xname": xname or "x"},
     )
+
+def sample_data(xs, ys, keys):
```

Comment:
Can you modify sample_data to use exponentially decaying weights instead of linear decaying